# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-mesh-pattern-rasterizer.c
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-mesh-pattern-rasterizer.c
--- a/gfx/cairo/cairo/src/cairo-mesh-pattern-rasterizer.c	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-mesh-pattern-rasterizer.c	Mon May 06 19:25:12 2024 +0000
@@ -659,7 +659,7 @@
  *        width, height are the dimensions of the image
  *        stride is the stride in bytes between adjacent rows
  *        vshift is log2(n) if n is the number of desired steps
- *        p[i][j], p[i][j] are the the nodes of the Bezier patch
+ *        p[i][j], p[i][j] are the nodes of the Bezier patch
  *        col[i][j] is the j-th color component of the i-th corner
  *
  * Output: data will be changed to have the requested patch drawn in